src/app/routes/index.test.jsx

```diff
@@ -362,78 +341,11 @@ describe('Routes', () => {
       ).toBeInTheDocument();
     });
 
-    it('should route to and render a photo gallery page', async () => {
```

Comment:
**Note for reviewers:** I have deleted this test photo gallery pages are now routed to the article page